### ISSUE TITLE
Extract data of courses

### ISSUE THREAD

**Issue by ofiralexi** [TechnionYP5779/team1#175]
Extract data of relevant courses:
- [ ]  course name and number
- [ ]  syllabus
- [ ]  lecture hours and locations
- [ ] Any more data you can get
into a CSV file

**Comment by ofiralexi** [comment]
Some helpful links
UG REP - 
http://ug3.technion.ac.il/rep/REPFILE.zip
PARSER -
https://github.com/lutzky/repy
hosted:
https://repy-176217.appspot.com/

**Comment by ShaiZeevi** [comment]
Indeed, please update how it is represented in DB, those using the data will decide how it should be read\spoken

**Comment by Matanpugach** [comment]
will be stored as a string, repesenting a logical connection: for example: 
'111111&222222|333333&444444'

**Comment by ofiralexi** [comment]
@ShaiZeevi can we close?
